### DISCUSSION TITLE
how to deal with Unpublished fields

### DISCUSSION THREAD

**Discussion by AchillesDougalis** [eprocurementontology/eprocurementontology#585] [Alignment]
The ePO and eForms teams need to align on how to represent unpublished fields.   In ePO 4.0.0, unpublished fields are represented as seen on the diagram below:
![GetImage (1)](https://github.com/OP-TED/ePO/assets/143995053/ecb8a406-f87b-4649-90b3-ec616602ab32)

**Comment by muricna** [comment]
Dear Achilles,

Again why is this marked as resolved when no solution is provided, if it is because it is moved to an issue then a reference to the issue needs to be made.

Kind regards,

Natalie

**Reply by AchillesDougalis** [reply]
Dear Natalie, there is a reference of the issue on the right, under "Events"
